app/models/katello/authorization/product.rb

```diff
@@ -19,7 +19,7 @@ def editable?
 
       def deletable?
         promoted_repos = repositories.select { |repo| repo.promoted? }
-        authorized?(:destroy_products) && promoted_repos.empty?
+        authorized?(:destroy_products) && (promoted_repos.empty? || Setting.find_by(name: 'delete_repo_across_cv')&.value == true)
```

Comment:
```suggestion
        authorized?(:destroy_products) && (promoted_repos.empty? || Setting[:delete_repo_across_cv])
```

Reply:
We should also use the above syntax to get the setting value in other places this is used.

Reply:
Updated the syntax. I'll keep this in mind in the future.